website/lychee.toml

```diff
@@ -8,4 +8,6 @@ max_retries = 3
 exclude = [
   # rate limited
   "https://epubs.siam.org/doi/10.1137/0108018",
+  # 403 for link checking only
+  "https://www.theblock.co",
```

Comment:
ignored now, there are some sites hostile to lick checking (DDOS prevention) that 403